diff --git a/src/jotai/mouse.ts b/src/jotai/mouse.ts
index c3279de..b68d61b 100644
--- a/src/jotai/mouse.ts
+++ b/src/jotai/mouse.ts
@@ -14,4 +14,4 @@ export const scrollDirectionAtom = atom(1);
 export const scrollIntervalAtom = atom(0);

 // mouse jiggler mode: enable or disable
-export const mouseJigglerModeAtom = atom<'enable' | 'disable'>('disable');
+export const mouseJigglerModeAtom = atom<'enable' | 'disable'>('enable');
